tests/unit/state/test_graph_combinatorics.py

```diff
@@ -0,0 +1,19 @@
+import pytest
+
+from expertsystem.state.particle import initialize_graph
+
+
+@pytest.mark.parametrize(
+    "final_state_groupings", [[[["pi0", "pi0"]]], [[["gamma", "pi0"]]],],
+)
+def test_initialize_graph(
+    final_state_groupings, dummy_topology, particle_database
+):
+    graphs = initialize_graph(
+        dummy_topology,
```

Comment:
Idea: the dummy topology is not even needed as a fixture. You could create a 3 body final state decay topology with the builder (there is only one topology).

```python
two_body_decay_node = InteractionNode("TwoBodyDecay", 1, 2)
simple_builder = SimpleStateTransitionTopologyBuilder(
    [two_body_decay_node]
￼)
all_graphs = simple_builder.build_graphs(1, 3)
```

Reply:
Mwa, I want to keep the tests for `Topology` isolated form the test for the topology builders.